data/mods/Dark-Skies-Above/README.md

```diff
@@ -1,16 +1,18 @@
 
 # Takeaway Features
-In short, Dark Skies is more difficult than vanilla DDA, and reccomended for players who have some experience under their belt. 
+In short, this Dark Skies is much much more difficult than vanilla DDA, and recommended for players who have some experience under their belt. 
 
- - Many more people were twisted into hostile mutants by alien bioweapons, now called Strays. They are slightly weaker than a vanilla zombie, but there come in larger groups and have a few nasty variants.
- - Alien soldiers occupy high-value targets and have deployed to the streets in order to reclaim the cities for resources. They can easily dispatch Strays, but are no more friendly to you and much more dangerous.
- - Many buildings were badly damaged during the Arrival and the ensuing conflict, destroying much of the supplies inside. 
- - Invasive alien species prowl the wilderness, some hostile.
- - WIP: Mutations and CBMs, among other sci-fi native to DDA, have been stripped out, with intent for similar tech to come from reverse engineering alien gear.
- - WIP: NPC missions, building up a resistance, working to secure small victories against the occupation forces.
- - WIP: Lots of 'dungeons' to explore and clever ways to strip down alien tech for parts to build tools, similar to Aftershock.
+ - Many more people were twisted into hostile mutants by alien bioweapons, now called Strays. They are stronger than a vanilla zombie, but there come in smaller groups, usually, and have a few nasty variants.
+ - Alien soldiers occupy high-value targets and have deployed to the streets in order to reclaim the cities for resources and neutralize threats. They each have uniquely debilitating psionic effects, a powerful melee weapon and an insidious dart gun. 
+ - Invasive alien species prowl the wilderness, some hostile some are not. 
+ - If you survive long enough, you may even see the Xenoid's emissaries to humanity.
+ 
+ -WIP Map: "Peach Trees" Arcology by Zenture Industries®
+ -WIP Map: The Alien Hive Structure
+ -WIP Map: The Xenoid Fortress
+ 
 
 # Meta
-Being a conversion mod, this mod won't work with most other mods. Likewise, tileset support is probably going to be patchy, however I'll be making good use of looks_like for your viewing pleasure.
+This is NOT a total conversion mod, I'll strive to make Dark Skies Above work as friendly with other mods as possible, and will try to maintain tileset support based off UndeadPeople tileset. I intend for this mod to challenge you as much as possible and won't pull any punches, this mod is people who want a crazy challenge and/or radical change to the base game. Having said that I really don't have any idea what I'm doing so keep that in mind before complaining. The original design document is included with the mod, put the specifics may not be included and should be considered apocryphal as far as future features. 
```

Comment:
You should probably remove the original design document if its not longer applicable to the current vision of the mod.